### DISCUSSION TITLE
German lemmatizer confused by capitalization

### DISCUSSION THREAD

**Discussion by giopina** [explosion/spaCy#9466] [Language Support]
I'm having issues with the lemmatizer for German (both in v3.0.6 and v2.3.2, using both `de_core_news_lg` and `de_dep_news_trf`).
Basically, the lemmatizer gets confused by the capitalization of the verb, and can't assign the right lemma to it (while the POS tagger is actually correct).

Code:
```
import spacy
nlp = spacy.load("de_core_news_lg")
doc = nlp('Meldet sich keiner')
for token in doc:
    print(token,token.lemma_,token.pos_,token.tag_)
```
Output:
```
Meldet Meldet VERB VVFIN
sich sich PRON PRF
keiner kein PRON PIS
```
(the lemma of _meldet_ should be _melden_)

Is there some general solution/fix to this issue?

_Originally posted by @giopina in https://github.com/explosion/spaCy/issues/2668#issuecomment-940885415_

**Comment by adrianeboyd** [comment]
The current extremely simple lookup lemmatizer is just not very good, a closely related discussion: https://github.com/explosion/spaCy/discussions/8695#discussioncomment-1243549. It doesn't know anything about POS, casing, or spelling variation.

We do have some good news to report, though: we have internal work-in-progress on a statistical lemmatizer that should be much, much better than the lookup lemmatizer, which was only meant to be a stopgap solution and has been the default for German for way too long at this point. With the new lemmatizer, the accuracy on TIGER is ~97%. Keep an eye out for an official announcement soon!
